src/main/java/seedu/linkedout/model/applicant/NameContainsAllKeywordsPredicate.java

```diff
@@ -28,4 +28,9 @@ public boolean equals(Object other) {
                 && keywords.equalsIgnoreCase(((NameContainsAllKeywordsPredicate) other).keywords)); // state check
     }
 
+    @Override
+    public int numberOfKeywordMatches(Applicant applicant) {
+        return 1;
```

Comment:
Building upon @zzhikai 's comment, perhaps you could declare 1 as a constant, so it wouldn't be a magic number.
And yup javadocs would help!

```suggestion
        return SINGLE_APPLICANT;
```

Reply:
Bump, consider this as well. Maybe something more descriptive if you have any ideas, ONE_UNIQUE_APPLICANT etc or it seems its about Keyword matches, so maybe ALL_KEY_WORDS.. seems abit odd